src/relay/blockchain/events_informations.py

```diff
@@ -71,6 +216,15 @@ def balance_viewed_from_user(balance_update_event):
         raise RuntimeError("Unexpected balance update event")
 
 
+def sorted_events(events, reverse=False):
+    def key(event):
+        if event.blocknumber is None:
+            return math.inf
+        return event.blocknumber
```

Comment:
I think it should matter. I will update it but I failed to write a test since it seems impossible to have two transactions from the same account in a block with eth_tester.